app/store/storage-wrapper.test.ts

```diff
@@ -0,0 +1,68 @@
+jest.unmock('./storage-wrapper');
```

Comment:
S: you could add a small comment here to explain why you unmock.
```suggestion
// Unmocking storage-wrapper as it's mocked in testSetup directly
// to allow easy testing of other parts of the app
// but here we want to actually test storage-wrapper
jest.unmock('./storage-wrapper');
```

Reply:
Thanks Nico! Yeah sounds great to explain why that was needed!